dev/docker-compose.yaml

```diff
@@ -117,8 +117,8 @@ services:
       - BRANDING_PATH=/var/cfg/branding.json
       - NOTIFY_INBOX_ONLY=true
       # - PARALLEL=2
-      # - INIT_SCRIPT_URL=https://raw.githubusercontent.com/hcengineering/init/main/script.yaml
-      # - INIT_WORKSPACE=onboarding
+      - INIT_SCRIPT_URL=https://raw.githubusercontent.com/hcengineering/init/main/script.yaml
+      - INIT_WORKSPACE=test
```

Comment:
Let's disable it for dev

Reply:
it is bad to disable, since we miss errors.